Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -560,6 +574,8 @@ internal AudioClip FindAudio(int id)
                                     Debug.Log("unknown asset: " + asset.name);
                                 break;
                         }
+
+                        SendEventData(new MeshSyncAnalyticsData() { asset_Type = asset.type });
```

Comment:
`SceneCache` is also using UpdateScene().

I think you might want to avoid confusing the use of MeshSyncServer and SceneCache ?

Reply:
This was fixed now, thanks for spotting it.